src/snapred/backend/dao/request/CalibrationAssessmentRequest.py

```diff
@@ -30,3 +31,7 @@ class CalibrationAssessmentRequest(BaseModel):
     nBinsAcrossPeakWidth: int = Config["calibration.diffraction.nBinsAcrossPeakWidth"]
     peakIntensityThreshold: float = Config["calibration.diffraction.peakIntensityThreshold"]
     peakType: ALLOWED_PEAK_TYPES = "Gaussian"
+    fwhmMultiplierLimit: Limit[float] = Limit(
```

Comment:
Something we may want to nip in the bud with this pr is treating the multipliers as a limit, could we change its type to:
```
class Pair(GenericModel, Generic[T]):
    value1: T
    value2: T
    
    def __init__(self, minimum: T = None, maximum: T = None, **kwargs):
        if minimum is not None and maximum is not None:
            super().__init__(value1=minimum, value2=maximum)
        else:
            super().__init__(**kwargs)
            
    @property
    def minimum(self) -> T:
        return self.value1
    
    @property
    def maximum(self) -> T:
        return self.value2
```

as an intermediate solution?